dll/win32/shell32/shlfileop.cpp

```diff
@@ -755,6 +755,21 @@ static DWORD SHNotifyMoveFileW(FILE_OPERATION *op, LPCWSTR src, LPCWSTR dest, BO
 #endif
 }
 
+static BOOL SHIsCdRom(LPCWSTR path)
+{
+    WCHAR tmp[] = { 'A',':','\\',0 };
```

Comment:
you may be able to get rid of this wine-style string?

Reply:
sure